src/mime.rs

```diff
@@ -1,5 +1,5 @@
 /// List of allowed mimes.
-static ALLOWED_MIME_TYPES: [&str; 2] = ["text/plain;charset=utf-8", "UTF8_STRING"];
+static ALLOWED_MIME_TYPES: [&str; 3] = ["text/plain;charset=utf-8", "text/plain", "UTF8_STRING"];
```

Comment:
Could we have this in the end of the list? As well as it being checked after the rest of the mimes?

Also, text/plain may be non-utf8, so we have to account for that and try `lossy` functions or something like that, though it's not a good idea... I guess we should just try to build a utf8 string and fail if it's not.